backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/ImagesHandler.java

```diff
@@ -963,6 +966,14 @@ public static long computeCowImageNeededSize(VolumeFormat sourceFormat, long act
                 : actualSize;
     }
 
+    public static long computeDestinationImageInitialSizeInBytes(long sourceImageSize) {
```

Comment:
since `sourceImageSize` refers to the virtual size, it would be the same for both the source and target images, so how about imageSize?

Reply:
Done.
Updated the method and variable names and also changed the method to receive `Image`.